recipes/wasmtime/all/conanfile.py

```diff
@@ -68,6 +74,8 @@ def validate(self):
             raise ConanInvalidConfiguration(
                 f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
             )
+        if self._is_glibc_older_than_2_28:
+            raise ConanInvalidConfiguration(f"{self.ref} requires glibc 2.28 or later.")
```

Comment:
Is glib 2.28 required for any older version too, or only 23.0.1? I guess something is missing, because this package was working before.

Reply:
@uilianries 
Thank you for your review.
You're right.
wasmtime 23.0.1 only requires glibc 2.28.
I fixed it.